pkg/storaged/utils.js

```diff
@@ -757,6 +884,37 @@
     return usage;
 }
 
+async function set_fsys_noauto(client, block, mount_point) {
+    for (const conf of block.Configuration) {
+        if (conf[0] == "fstab" &&
+            decode_filename(conf[1].dir.v) == mount_point) {
+            const options = parse_options(get_block_mntopts(conf[1]));
+            if (options.indexOf("noauto") >= 0)
+                continue;
+            options.push("noauto");
+            const new_conf = [
+                "fstab",
+                Object.assign({ }, conf[1],
```

Comment:
These 11 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19388-20230926-135906-360ec0bd-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)